docs/source/api.rst

```diff
@@ -123,6 +126,8 @@ Metrics
   .. automodule:: cuml.metrics.cluster.entropy
     :members:
 
+  .. automodule:: cuml.metrics._ranking
```

Comment:
Usually we don't want to expose modules that start with an underscore. roc_auc_score should also be available as cuml.metrics.roc_auc_score.
I think you just want to hit it with an automethod on cuml.metrics.roc_score instead?

Reply:
Will fix this shortly and resubmit.